Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -490,13 +493,15 @@ extension NIOSSLCertificate: CustomStringConvertible {
             let commonName = String(decoding: commonNameBytes, as: UTF8.self)
             desc += ";common_name=" + commonName
         }
-        if let alternativeName = self.subjectAlternativeNames() {
-            let altNames = alternativeName.map { name in
-                switch name {
+        if let alternativeName = self._subjectAlternativeNames() {
+            let altNames = alternativeName.compactMap { name in
+                switch try? name.get() {
```

Comment:
Should we swallow or throw here? The previous implementation did swallow the error deeper in the stack.